src/pages/end.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+const EndPage = () => {
+  return (
+    <main className="pt-4 container text-center">
+      <h1 className="text-xl">Done!</h1>
```

Comment:
We need to add the score and the replay button

Reply:
Done!